doc/pages/architecture/030-interface.md

```diff
@@ -104,11 +133,14 @@ This is how a parametrized mapping looks like:
         },
     [...]
 ```
+
 In this example, `/0/value`, `/1/value` or `/test/value` all map to a valid endpoint, while
 `/te/st/value` can't be resolved by any endpoint.
 
 ### Supported data types
+
 The following types are supported:
+
 * `double`: A double-precision floating-point format as specified by binary64, by the IEEE 754
```

Comment:
* `double`: A double-precision floating-point number as specified by binary64, by the IEEE 754 (NaNs and other non numerical values are not supported).